pkg/nuget/lock/parse_test.go

```diff
@@ -52,6 +52,12 @@ func TestParse(t *testing.T) {
 				return ret < 0
 			})
 
+			for _, lib := range got {
+				sort.Slice(lib.Locations, func(i, j int) bool {
```

Comment:
What if sorting locations in `Parse`?

Reply:
hmm.. that's a good idea. 
Sorted locations will be convenient for the user's work.
I will add it.

Reply:
Done!